components/esp_http_client/esp_http_client.c

```diff
@@ -11,8 +11,11 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+//
+// Modifications copyright 2019 Tricot Inc.
```

Comment:
Typically additional notice is required by the Apache license, though the license grant in the contributor agreement is broader. (That said, it's a license grant, not a copyright assignment.)